### PR TITLE
[pack-script] Update pack script to replace the make_apk with app-tool.

### PR DESCRIPTION
**Pull request by haoyunfeix**
1. Keep the old usage for:
    apk-ext-opt
    apk-cmd-opt
    apk-common-opts
    apk-pkg-opt
    apk-fullscreen-opt
    apk-url-opt
    apk-version-opt
    apk-icon-opt
2. Add new support for:
    apk-orientation-opt
    apk-screenOn-opt
    apk-animatableView-opt
    apk-webp-opt
    apk-shortName-opt
    apk-permissions-opt

BUG=XWALK-5277 XWALK-5727 XWALK-5728 XWALK-5735

### CODE REVIEW

File: tools/build/build_android.py [hunk 1]
```diff
@@ -56,119 +56,215 @@ def packAPK(build_json=None, app_src=None, app_dest=None, app_name=None):
     BUILD_ROOT = varshop.getValue("BUILD_ROOT")
     BUILD_ROOT_SRC = varshop.getValue("BUILD_ROOT_SRC")
     BUILD_TIME= varshop.getValue("BUILD_TIME")
+    CROSSWALK_VERSION = varshop.getValue("CROSSWALK_VERSION")
```
Comment (haoxli) [inline tools/build/build_android.py]: 100644 -> 100755

Comment (haoyunfeix) [inline tools/build/build_android.py]: Fixed

**Comment by Honry** [comment]
LGTM

**Comment by jennycao** [comment]
LGTM

**Pull request merged**